setup.php

```diff
@@ -81,6 +84,45 @@ function plugin_init_uninstall() {
             }
          }
 
+         $PLUGIN_HOOKS['pre_item_update']['uninstall'] = [
+             'Config' => ['PluginUninstallConfig', 'preConfigSet']
+         ];
+          $PLUGIN_HOOKS['pre_item_add']['uninstall'] = [
+              'Config' => ['PluginUninstallConfig', 'preConfigSet']
+          ];
+
+         $PLUGIN_HOOKS[Hooks::STALE_AGENT_CONFIG]['uninstall'] = [
+             [
+                 'label' => 'Apply uninstall profile',
+                 'item_action' => true,
+                 'render_callback' => static function ($config) {
+                     $stale_agents_uninstall = Config::getConfigurationValue('plugin:uninstall', 'stale_agents_uninstall');
+                     if (!\PluginUninstallModel::canView()) {
+                         return false;
+                     }
+                     return \PluginUninstallModel::dropdown([
+                         'name' => '_stale_agents_uninstall',
+                         'value' => $stale_agents_uninstall ?? 0,
+                         'entity' => $_SESSION['glpiactive_entity'],
+                         'condition' => [
+                             'types_id' => \PluginUninstallModel::TYPE_MODEL_UNINSTALL
+                         ],
+                         'display' => false,
+                     ]);
+                 },
```

Comment:
This function should be transformed into a class method and referenced using a callable syntax like `'PluginUninstallConfig::renderStaleAgentConfigField'`.
`setup.php` files should contains less code as possible.